sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/ConfigurationManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    internal class ConfigurationManager
+    {
+        private readonly string[] SERVICE_IDs = new string[] { "99045fe1-7639-4a75-9d4a-577b6ca3810f" };
```

Comment:
How stable is this value?    Most of the things in here seem like we should consider having config or environment variable overrides.

Reply:
Allot of these are fixed, but there are a few that are overridable via app settings or environment vars. Listed are the overridables.

` internal static bool BypassValidation => GetConfigValue(BYPASS_VALIDATION, false);
        internal static bool EZAuthEnabled => GetConfigValue(EZAUTH_ENABLED, false);
        internal static string CallerAppId => GetConfigValue(CUSTOM_CALLER_APPID, null);
        internal string TenantId => GetConfigValue(TENANT_ID, triggerAttribute.TenantId);
        internal string AudienceAppId => GetConfigValue(AUDIENCE_APPID, triggerAttribute.AudienceAppId);
        internal string OpenIdConnectHost => triggerAttribute.OpenIdConnectHost;`